### ISSUE TITLE
List branches in MRU order

### ISSUE THREAD

**Issue by thomaseizinger** [felipec/git-completion#10]
I'd like to reboot https://github.com/ohmyzsh/ohmyzsh/issues/8767 and get this properly upstreamed here. Would you accept a contribution for this? Should this be configurable?

**Comment by felipec** [comment]
Yes, that would be a useful feature and I would accept the patches if the implementation doesn't turn out too complicated. Unfortunately I have a feeling it might be more complicated than it looks like.

We'll see.
